scripts/main/albums.js

```diff
@@ -55,14 +55,10 @@ albums.load = function() {
 
 albums.parse = function(album) {
 
-	if (album.password==='1' && lychee.publicMode===true) {
```

Comment:
You're going to make me go think about `publicMode` now, aren't you...

Reply:
Doesn't seem to be important (here)

Reply:
`publicMode` tests are generally insufficient in v4, where in addition to v3-style public and logged in (Admin in v4) modes, there are also non-admin users, album sharing between users, and accounts with upload capability disabled. I'm planning to address this issue comprehensively in a future PR but for now I do it case-by-case if it happens to affect the code I work on anyway. In this case, as you say, the `publicMode` case is not required.